cpp/vm/evmzero/interpreter.h

```diff
@@ -54,7 +55,7 @@ struct InterpreterArgs {
   const evmc_message* message = nullptr;
   const evmc_host_interface* host_interface = nullptr;
   evmc_host_context* host_context = nullptr;
-  evmc_revision revision = EVMC_ISTANBUL;
+  evmc_revision revision = EVMC_SHANGHAI;
```

Comment:
This seems unrelated to the intent of this PR. What is the motivation for this?

Reply:
I see a number of issues with this member, I will revert it to Istanbul until I come out with a better method to provide better revision testing for evmzero (other than CT) here: #478

Some notes about the issue: 

- The default value for revision in the context struct is always overwritten, either by the effective InterpreterArgs, or by the unit test Args: This violates the single point of truth. 
- evmzero lacks of the definition for the range of supported revisions. (an equivalent to `newestSupportedRevision` in lfvm), but instead provides a baseline revision by default.  This works in conjunction with unit test running under the assumption that instructions follow Istanbul semantics unless another revision is prescribed. This unit test fails to model semantic changes that happen for some instructions between revisions. IMO it would be beneficial to explicitly test every revision support for every instruction in the unit test level as well.

Reply:
Reverted, we will come back to this issue some other time.